### PR TITLE
Bump pino dependency to pino 6 and switch to Github Actions

### PR DESCRIPTION
**Pull request by ShogunPanda**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [6.x, 8.x, 10.x, 12.x, 13.x]
```
Comment (mcollina) [inline .github/workflows/ci.yml]: The minimum version supported by pino@6 is node 10.

File: Readme.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
-# pino-multi-stream &nbsp;[![Build Status](https://travis-ci.org/pinojs/pino-multi-stream.svg?branch=master)](https://travis-ci.org/pinojs/pino-multi-stream)
+# pino-multi-stream &nbsp;[![Build Status](https://badgen.net/github/status/pinojs/pino-multi-stream/master/ci)](https://github.com/pinojs/pino-multi-stream/actions?query=workflow%3ACI)
```
Comment (jsumners) [inline Readme.md]: We're using https://shields.io/ in the core `pino` repo. I think it would be good to be consistent and use it here as well.

**Comment by jsumners** [comment]
Please disable prettier when working in this repository.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ShogunPanda) [reply .github/workflows/ci.yml]: Good point. I blindly copied the travis file. Removed now.

**Comment by ShogunPanda** [comment]
Yeah, that's exactly what messed the Readme. I have it enabled globally on my VS Code. I've disabled it now and regenerated the Readme.

File: Readme.md (hunk 2 above)
Comment (ShogunPanda) [reply Readme.md]: Sure, will change here and in pino-pretty#101

Comment (ShogunPanda) [reply Readme.md]: Done now!

**Pull request merged**